app/javascript/src/hyacinth_v1/components/digital_objects/search/facet_sidebar/FacetDropdown.js

```diff
@@ -13,9 +13,9 @@ const FacetDropdown = ({ facet, onFacetSelect, selectedValues }) => {
 
   return (
     <div className="border-bottom border-secondary">
-      <Button className="px-0" variant="link" block onClick={() => setOpen(o => !o)}>
-        <span className="text-left float-left">{facet.displayLabel}</span>
-        <span className="float-right"><FontAwesomeIcon size="lg" icon={open ? 'caret-up' : 'caret-down'} /></span>
+      <Button className="px-0 w-100" variant="link" onClick={() => setOpen(o => !o)}>
+        <span className="float-start">{facet.displayLabel}</span>
```

Comment:
should there be a `text-start` class here to replace the previous `text-left`?

Reply:
👍